src/lib/validators/validateFullName.js

```diff
@@ -2,9 +2,9 @@
 export const validateFullName = (fullName: string) => {
   if (fullName.length === 0) {
     return ERROR_MESSAGE.EMPTY
-  } else if (fullName.match(/\d+/) !== null) {
+  } else if (fullName.match(/\d+/) != null) {
```

Comment:
should stay != null
== null will return true on both null and undefined
=== null true only on null

Reply:
The most correct way:

```suggestion
  } else if (/\d+/.test(fullName)) {
```